### PR TITLE
chore(vg_lite_tvg): fix build warnings

### PR DESCRIPTION
**Pull request by W-Mai**
### Description of the feature or fix

```cpp
lvgl/src/others/vg_lite_tvg/vg_lite_tvg.cpp:1738:62: warning: unused parameter 'paint_color' [-Wunused-parameter]
 1738 |                                              vg_lite_color_t paint_color,
      |                                                              ^
lvgl/src/others/vg_lite_tvg/vg_lite_tvg.cpp:1740:63: warning: unused parameter 'filter' [-Wunused-parameter]
 1740 |                                              vg_lite_filter_t filter)
      |                                                               ^

```

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by FASTSHIFT — DISMISSED** [review]

**Comment by XuNeo** [comment]
Let's try to add warning flag to file `vg_lite_tvg.cpp` only

File: env_support/cmake/custom.cmake [hunk 1]
```diff
@@ -51,6 +51,8 @@ if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
     target_link_libraries(lvgl_thorvg PUBLIC lvgl)
 endif()
 
+set_source_files_properties(${LVGL_ROOT_DIR}/src/others/vg_lite_tvg/vg_lite_tvg.cpp PROPERTIES COMPILE_FLAGS -Wunused-parameter)
```
Comment (kisvegabor) [inline env_support/cmake/custom.cmake:54]: From [Cmake docs](https://cmake.org/cmake/help/latest/command/set_source_files_properties.html):
> By default, source file properties are only visible to targets added in the same directory (CMakeLists.txt). 

So unless I miss something this line has no effect.

Comment (XuNeo) [reply env_support/cmake/custom.cmake:54]: I tested it and the warnings are correctly reported after adding this line. I guess it means we cannot add file properties to files added from another CMakeLists.txt.

Comment (FASTSHIFT) [reply env_support/cmake/custom.cmake:54]: Can we add checks to all C++?

Comment (kisvegabor) [reply env_support/cmake/custom.cmake:54]: Okay then.

**Review by kisvegabor — APPROVED** [review]

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**